lib/tasks/create_popular_links_and_link_to_homepage.rake

```diff
@@ -0,0 +1,14 @@
+desc "Create popular links and links to homepage"
+task create_popular_links_and_link_to_homepage: [:environment] do
+  homepage_content_id = "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a".freeze
+  popular_links = PopularLinksEdition.new(title: "Homepage popular links", link_items: [{ url: "url1.com", title: "title1" }, { url: "url2.com", title: "title2" }, { url: "url3.com", title: "title3" }, { url: "url4.com", title: "title4" }, { url: "url5.com", title: "title5" }, { url: "url6.com", title: "title6" }])
```

Comment:
Is it worth populating with the current popular link values?

Reply:
I am not sure about this, the values can change between when the code is merged and when the rake task is run.

Reply:
The team has decided that we will leave as these defaults now, and we can then guide a content designer through the process of updating to use the current values that are on the homepage.